lib/index.js

```diff
@@ -251,7 +251,7 @@ class Peco {
   }
 
   resolvePecoDir(...args) {
-    return path.join(this.options.baseDir, '.peco', ...args)
+    return path.posix.join(this.options.baseDir, '.peco', ...args)
```

Comment:
maybe `path.normalize(path.join(this.options.baseDir, '.peco', ...args))` instead 🤔

Reply:
hmmm...  `path.normalize`  not working for me, it's still return `foo\bar\` in windows 😄